src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,132 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * Class handling List of Tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> list;
+    private boolean isDone;
+
+    /**
+     * Constructor method for TaskList.
+     */
+    public TaskList() {
+        list = new ArrayList<>();
+        isDone = false;
+    }
+
+    /**
+     * Gets the ArrayList of Tasks.
+     * @return ArrayList of Tasks
```

Comment:
Could add empty line between description and parameters section, here and in other places